pkg/bpf/bpf.go

```diff
@@ -264,6 +273,87 @@ func recoverVersionMap(pinPath string) *ebpf.Map {
 	return versionMap
 }
 
+func (l *BpfLoader) skipSpecificProbe() {
+	nodeName := os.Getenv("NODE_NAME")
+	if nodeName == "" {
+		log.Errorf("skip kubelet probe failed: %s", "node name empty")
+		return
+	}
+
+	clientSet, err := utils.GetK8sclient()
+	if err != nil {
+		log.Errorf("get kubernetest client for getting node IP error: %v", err)
+	}
+	nodeIP := getNodeIPAddress(nodeName, clientSet)
+
+	gateway := calculateGateway(getNodePodSubGateway(nodeName, clientSet))
+
+	keyOfNodeIP := uint32(1)
+	keyOfPodSubGateway := uint32(2)
+	if l.kmeshConfig == nil {
+		log.Errorf("skip kubelet probe failed: %v", "kmeshConfigMap is nil")
+		return
+	}
+	if err := l.kmeshConfig.Update(&keyOfNodeIP, &nodeIP, ebpf.UpdateAny); err != nil {
+		log.Errorf("update nodeIP in kmeshConfigMap failed: %v", err)
+		return
+	}
+	if err := l.kmeshConfig.Update(&keyOfPodSubGateway, &gateway, ebpf.UpdateAny); err != nil {
+		log.Errorf("update sub gateway of pod in kmeshConfigMap failed: %v", err)
+		return
+	}
+}
+
+func getNodeIPAddress(nodeName string, clientSet kubernetes.Interface) uint32 {
+	node, err := clientSet.CoreV1().Nodes().Get(context.TODO(), nodeName, metav1.GetOptions{})
+	if err != nil {
+		log.Errorf("failed to get node: %v", err)
+		return uint32(0)
+	}
+
+	var nodeIPStr string
+	nodeAddresses := node.Status.Addresses
+	for _, address := range nodeAddresses {
+		if address.Type == corev1.NodeInternalIP {
+			nodeIPStr = address.Address
+		}
+	}
+
+	nodeIP := net.ParseIP(nodeIPStr)
+	return binaryToUint32(nodeIP[12:16])
```

Comment:
why only make use the last 4 bytes

Reply:
we have `ConvertIpToUint32` and `ConvertPortToBigEndian` select which suitable

Reply:
> why only make use the last 4 bytes

Since the type of the map value is `unit32`, only the ipv4 case was considered first.